gulpfile.coffee

```diff
@@ -27,7 +27,7 @@ gulp.task 'teacup', ->
     .pipe rename extname: '.html'
     .pipe gulp.dest options.destination
 
-gulp.task 'watch', ->
+gulp.task 'watch', ['teacup'], ->
```

Comment:
Now `teacup` task runs before `watch` task starts.

Since `watch` task runs before `serve` task starts (see line 35), in effect when you can execute `gulp serve` and get up to date files served - bingo!